packages/eas-cli/src/commands/update/index.ts

```diff
@@ -208,28 +169,27 @@ export default class UpdatePublish extends EasCommand {
       description: `File containing the PEM-encoded private key corresponding to the certificate in expo-updates' configuration. Defaults to a file named "private-key.pem" in the certificate's directory.`,
       required: false,
     }),
-    'non-interactive': Flags.boolean({
-      default: false,
-      description: 'Run command in non-interactive mode',
-    }),
+    // do not expose limit/offset since they don't make sense on this command
```

Comment:
that comment is redundant, a lot fo commands do not have pagination and are non-interactive or can provide json output. Those options are not related to each other. Maybe consider rethinking how EasPaginatedQueryFlags looks and separating those.